folsom-opentelemetry-metrics/pom.xml

```
@@ -0,0 +1,38 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>com.spotify</groupId>
+    <artifactId>folsom-parent</artifactId>
+    <version>1.16.2-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>folsom-opentelemetry-metrics</artifactId>
+
+  <dependencies>
+    <dependency>
+      <groupId>com.spotify</groupId>
+      <artifactId>folsom</artifactId>
+      <version>${project.version}</version>
+      <scope>provided</scope>
+    </dependency>
+    <dependency>
+      <groupId>io.opentelemetry</groupId>
+      <artifactId>opentelemetry-sdk-metrics</artifactId>
```

You don't need to depend on the SDK, you're only using the OTel API in this PR:
```suggestion
      <artifactId>opentelemetry-api</artifactId>
```